cpp/modmesh/pilot/R3DWidget.cpp

```diff
@@ -84,8 +84,31 @@ void R3DWidget::updateWorld(std::shared_ptr<WorldFp64> const & world)
             child->deleteLater();
         }
     }
+
     new RVertices(world, m_scene);
     new RLines(world, m_scene);
+
+    auto vertices = std::make_shared<RVertices>(world, m_scene);
+    auto lines = std::make_shared<RLines>(world, m_scene);
+
+    float box_minX = std::min(vertices->bounding_min().x(), lines->bounding_min().x());
```

Comment:
Naming convention: `snake_case_x`.

Please add code comments to explain what the calculation does.  It is not crystal clear by simply reading the code.